chacra/async/__init__.py

```diff
@@ -107,6 +110,7 @@ def create_deb_repo(repo_id):
     # updated so we prevent piling up if other binaries are being posted
     repo.path = paths['absolute']
     repo.is_updating = True
+    repo.is_queued = False
```

Comment:
I would prefer doing this at the very end (line 201 for this task) because the processing of this repo may take a long time. Sufficiently long that the queue will get pilled on again while processing. 

It wouldn't be as much as we currently have but delaying the `is_queued` property will make it even leaner